dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaAPIHostsTask.java

```diff
@@ -54,16 +58,25 @@ protected Void doRun(
           "Cloudera clusters must be initialized before hosts dumping.");
     }
 
+    List<ClouderaHostDTO> hosts = new ArrayList<>();
     try (Writer writer = sink.asCharSink(StandardCharsets.UTF_8).openBufferedStream()) {
       for (ClouderaClusterDTO cluster : clusters) {
         String hostPerClusterUrl = handle.getApiURI() + "/clusters/" + cluster.getName() + "/hosts";
 
-        try (CloseableHttpResponse hosts = httpClient.execute(new HttpGet(hostPerClusterUrl))) {
-          JsonNode jsonNode = objectMapper.readTree(hosts.getEntity().getContent());
-          writer.write(jsonNode.toString());
+        ApiHostListDto apiHosts;
```

Comment:
the logic with this variable can be moved under `try`.  So the variable will be local.

Reply:
Fixed